index.src.html

```diff
@@ -1743,7 +1765,11 @@ <h4 algorithm id="construct-federatedcredential-data">
   2.  Define appropriate {{Credential/[[CollectFromCredentialStore]](options)}},
       {{Credential/[[DiscoverFromExternalSource]](options)}}, and
       {{Credential/[[Store]](credential)}} methods on `ExampleCredential`'s
-      [=interface object=]:
+      [=interface object=]. {{Credential/[[CollectFromCredentialStore]](options)}}
+      is appropriate for [=credentials=] that remain [=effective=] forever and
+      can therefore simply be copied out of the [=credential store=], while
+      {{Credential/[[DiscoverFromExternalSource]](options)}} is appropriate for
+      [=credentials=] that need to be re-generated from a [=credential source=].
```

Comment:
As an aside, I think you're right that we haven't gotten this division right yet. I think the central "do we have the thing locally in the credential store?" vs "do I have to ask something else?" is important, but the way we're using it in the `get()` algorithm doesn't feel right yet.

Thoughts welcome...